src/API/IJobStore.cs

```diff
@@ -51,7 +51,7 @@ public interface IJobStore : IHostedService
         /// The default implementation blocks the call until a pending request is available for submission.
         /// </summary>
         /// <param name="cancellationToken">cancellation token used to cancel the action.</param>
-        /// <returns><cr ="InferenceJob"/></returns>
+        /// <returns><see cref="Nvidia.Clara.DicomAdapter.API.InferenceJob"/></returns>
```

Comment:
the `<returns>...</returns>` tag is of very limited value. Intellisense doesn't even honor it - you should feel free to omit it in the future.

Reply:
It links correctly in DocFX.